### ISSUE TITLE
[Feature Request] Improved ARC search in Download panel / integration of metadata registry 

### ISSUE THREAD

**Issue by Brilator** [nfdi4plants/ARCitect#90]
User feedback: 

> Filter in ARCitect, filter repositories needed 

The current "download ARC" panel alphabetically lists all available ARCs. 
It would help to have a search-engine backed solution to filter for and find ARCs to select from, e.g.: 

- "show only my private ARCs"
- "show ARCs of my (DataHUB) group" 
- "show only public ARCs"
- "show all ARCs that work on barley"

This basically boils down to an integration of the metadata registry https://github.com/nfdi4plants/arcRegistry

**Comment by Brilator** [comment]
@gdoniparthi

**Comment by Brilator** [comment]
https://arcregistry.nfdi4plants.org/arcsearch

**Comment by Brilator** [comment]
Can we revive this discussion, try to keep things simple first and build search only based on what gitlab api provides?

**Comment by Brilator** [comment]
If easily possible, I'd suggest to 

- [ ] (allow to) sort the presented table similar to the options in DataHUB
- [ ] add filter / search fields
  - [ ] for DataHUB user name (displayed name and user short name)
  - [ ] namespace / DataHUB group
  - [ ] private / public

**Comment by JonasLukasczyk** [comment]
I will implement a searchbar similar to the one from the DataHUBs. For now deep searches (e.g., by ontology terms) will not be supported until the integration of the ARCregistry.

**Comment by Brilator** [comment]
¥

**Comment by Brilator** [comment]
Yes, DataHUB-like search functionality (based on gitlab metadata) would suffice for now and help a lot
